README.md

```diff
@@ -3,29 +3,69 @@ Scripts for easy backup and restore of Docker volumes
 
 ## Usage
 
-```
+```bash
 ./docker_volume_backup.sh {compose_file_path} {project_name} {backup_path} {backup_or_restore} {restore_date}
+# or
+./docker_full_backup.sh {compose_file_path} {project_name} {backup_path} {backup_or_restore} {restore_date}
 ```
 
 ## Examples
 
+### docker_volume_backup.sh
+
 Backup
 
-```
+```bash
 ./docker_volume_backup.sh /home/kiview/Gitlab/docker-compose.yml gitlab $(pwd)/backup backup
 ```
 
 Restore
 
-```
+```bash
 ./docker_volume_backup.sh /home/kiview/Gitlab/docker-compose.yml gitlab $(pwd)/backup restore 2016-10-19
 ```
+
+### docker_full_backup.sh
+
```

Comment:
Missing introductory text which explains, that the following code should reside in its own bash script.